static/app/components/gridEditable/styles.tsx

```diff
@@ -266,12 +267,7 @@ export const GridResizer = styled('div')<{dataRows: number}>`
 
   height: ${p => {
     const numOfRows = p.dataRows;
-    let height = GRID_HEAD_ROW_HEIGHT + numOfRows * GRID_BODY_ROW_HEIGHT;
-
-    if (numOfRows >= 1) {
-      // account for border-bottom height
-      height += numOfRows;
-    }
```

Comment:
`GRID_BODY_ROW_HEIGHT` already includes the border height (since we globally set `box-sizing: border-box;`), so we don't need to account for borders here.